### PR TITLE
Updated dependency

### PR DESCRIPTION
**Pull request by iEmiya**
Make necessary changes to build project under .NET Framework 4.6.1 & .Net Core 2.1, as well as updated links:
* Google.Protobuf 3.5.1 to 3.10.0
* Grpc 1.12.0 to 2.24.0

### CODE REVIEW

File: src/OpenTracing.Contrib.Grpc/netcore/OpenTracing.Contrib.Grpc.netCore.csproj [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    <IsPackable>true</IsPackable>
+    <Description>Adds OpenTracing instrumentation for .NET Core apps that use GRPC.</Description>
+    <PackageTags>opentracing;distributed-tracing;tracing;grpc;netcoreapp;dotnet</PackageTags>
+    <PackageId>OpenTracing.Contrib.Grpc</PackageId>
+    <Product>OpenTracing.Contrib.Grpc</Product>
+    <Authors>Benjamin Krämer, Medialogia®</Authors>
+    <AssemblyVersion>0.2.0.0</AssemblyVersion>
+    <FileVersion>0.2.0.0</FileVersion>
+    <PackageProjectUrl>https://github.com/Medialogia/csharp-grpc</PackageProjectUrl>
+    <RepositoryUrl>git://github.com/Medialogia/csharp-grpc</RepositoryUrl>
```
Comment (Falco20019) [inline src/OpenTracing.Contrib.Grpc/netcore/OpenTracing.Contrib.Grpc.netCore.csproj:14]: This should point to https://github.com/opentracing-contrib/csharp-grpc

**Review by Falco20019 — CHANGES_REQUESTED** [review]
Why was it necessary to change it into two projects? This project should only target .NET Standard and neither .NET Framework nor .NET Core to be also usable with Xamarin. Your changes would break this compatability.

If there is something limiting you due to .NET Standard 1.5 (in respect to dependencies) I would suggest upgrading it to .NET Standard 2.0 instead as this is still allowing usage with .NET Framework 4.6.1+ and .NET Core 2.0+ according to https://github.com/dotnet/standard/blob/master/docs/versions/netstandard2.0.md

Since this is a library, we also should use only the minimum required version of the dependencies since this is a transitive minimum. The application using the NuGET package can always use newer version. As long as Google.Protobuf 3.10.0 has no breaking changes over 3.5.1, it should not be updated in a library. We definetly need to update Grpc to 2.23.0 as minimal requirement.

Please let me know your requirements and maybe it's better to create an issue ticket with those to see what's the best solution.

**Comment by iEmiya** [comment]
> Why was it necessary to change it into two projects? This project should only target .NET Standard and neither .NET Framework nor .NET Core to be also usable with Xamarin. Your changes would break this compatability.
> 
> If there is something limiting you due to .NET Standard 1.5 (in respect to dependencies) I would suggest upgrading it to .NET Standard 2.0 instead as this is still allowing usage with .NET Framework 4.6.1+ and .NET Core 2.0+ according to https://github.com/dotnet/standard/blob/master/docs/versions/netstandard2.0.md
> 
> Since this is a library, we also should use only the minimum required version of the dependencies since this is a transitive minimum. The application using the NuGET package can always use newer version. As long as Google.Protobuf 3.10.0 has no breaking changes over 3.5.1, it should not be updated in a library. We definetly need to update Grpc to 2.23.0 as minimal requirement.
> 
> Please let me know your requirements and maybe it's better to create an issue ticket with those to see what's the best solution.

In our projects we use projects under .NET Framework 4.6.1 and .NET Core 2.1.
I think, it would be logical to add to assembly under .NET Standard 2.0, but assembly under .NET Framework 4.6.1 gets rid of significant amount of dependent DLL platform during installation and deployment of Windows Service.

**Comment by Falco20019** [comment]
Understood :) For greater compatibility I will add .NET Framework 4.5 DLL to NuGET in #15 which should be usable without any additional DLLs in 4.6.1

For minimal requirements I want to target Grpc 2.23.1 instead of 2.24.x. Due to a [bug](https://github.com/grpc/grpc/issues/20782) there is currently a problem when your project targets 2.24 and the library targets 2.23.1. There is already a [fixing attempt](https://github.com/grpc/grpc/pull/20859) posted that I will try out on Monday once back in office. I hope to fix this if it‘s working.

**Comment by pcwiese** [comment]
what happened with this change?

**Comment by Falco20019** [comment]
I haven’t had time lately for this project. Will try to look into it next week since the blocker should be resolved for some time now. Thanks for bringing it to my attention again.

**Comment by Falco20019** [comment]
Closed because superseeded by #13

**Pull request closed without merging**